sections/main-footer.liquid

```diff
@@ -9,52 +9,58 @@
     --yc-footer-text-color: {{ section.settings.footer_text_color.hex }};
     --yc-footer-copyright-background: {{ section.settings.footer_copyright_background.hex }};
   }
+
+  .toggle-footer {
+    display: {{ section.settings.toggle_footer ? 'none' : 'block' }};
+  }
 {% endstyle %}
 
-<footer class='yc-footer'>
-  <div class="columns container">
-    <div class="column logo">
-      <img src="{{ section.settings.footer_logo.src }}">
-    </div>
+{% if section.settings.toggle_footer != true %}
+  <footer class='yc-footer'>
+    <div class="columns container">
+      <div class="column logo">
+        <img src="{{ section.settings.footer_logo.src }}">
+      </div>
 
-    {%- for linklist in link-lists -%}
-      {%- if linklist.links | count > 0  and linklist != main-link-list -%}
-        <div class="column">
-          <div class="links">
-            <h1 class="title">{{ linklist.title }}</h1>
+      {%- for linklist in link-lists -%}
+        {%- if linklist.links | count > 0  and linklist != main-link-list -%}
+          <div class="column">
+            <div class="links">
+              <h1 class="title">{{ linklist.title }}</h1>
 
-            {%- for item in linklist.links -%}
-              <a class="item" href="{{ item.url }}">{{ item.title }}</a>
-            {%- endfor -%}
+              {%- for item in linklist.links -%}
+                <a class="item" href="{{ item.url }}">{{ item.title }}</a>
+              {%- endfor -%}
+            </div>
           </div>
+        {%- endif -%}
+      {%- endfor -%}
+        <div class="column social-media">
+          {%- for block in section.blocks -%}
+            {%- if block.settings.social_media_icon_image.src -%}
+              <a class='social-icon' href="{{ block.settings.social_media_url }}" target="_blank">
+                <img loading='lazy' class='image' src="{{ block.settings.social_media_icon_image.src }}" alt='social links'>
+              </a>
+              {% else %}
+              <a class='social-icon' href="{{ block.settings.social_media_url }}" target="_blank">
+                <img loading='lazy' src='{{ 'default_product.jpeg' | asset_url }}' alt='image placeholder'>
+              </a>
+            {%- endif -%}
+          {%- endfor %}
+        </div>
```

Comment:
Can you please make this `div` aligned with the `{%- endfor -%}`
<img width="342" alt="Screenshot 2023-05-03 at 09 54 46" src="https://user-images.githubusercontent.com/114928221/235872453-de187a42-12ff-4f8a-a18a-676b4d538759.png">